run.js

```diff
@@ -14,8 +16,10 @@ const defaultTemplate = require('./lib/default-template')
     const labels = list(core.getInput('labels'))
     const issueTemplate = core.getInput('issueTemplate')
     const createWithin = core.getInput('createWithin')
-    const agendaLabel = core.getInput('agendaLabel')
+    const agendaLabel = core.getInput('agendaLabel') || 'meeting-agenda'
```

Comment:
I think you can set this as a default for the action instead of here, right? The benefit being it would show in the action docs.

Reply:
Hm, I see it is set as a default above.  Are both necessary?

Reply:
No idea, this was my first time working on an actual Action (instead of consuming one), so I wanted to be extra safe 😅. It's probably not necessary though.

Reply:
Yep, setting it as the default would be ideal.